coordinator/src/main/java/org/apache/uniffle/coordinator/SimpleClusterManager.java

```diff
@@ -99,6 +101,13 @@ void nodesCheck() {
           }
         }
       }
+      if (!deleteIds.isEmpty() || outputAliveServerCount % 30 == 0) {
+        LOG.info("Alive servers number: {}, ids: {}",
```

Comment:
Why do we need this? We have metrics tell us how many there are alive servers.

Reply:
Number is not enough. I want to know which servers are lost from the coordinator side. 

And the number shown in log will be easy to find.

Reply:
Ok, I got it. Could `30` be configOption?

Reply:
Updated. 
Changelog
1. Introduce `rss.coordinator.server.periodic.output.interval.times`, default value = 30, this means it will output every 5min.

Reply:
Why do you judge `deleteIds.isEmpty()`?

Reply:
when deleteIds is not empty, it means the server list changed, we should output the latest alive servers.

Reply:
Do we want to use the two condition `deleteIds.isEmpty()` and `outputAliveServerCount % 30 == 0` at the same time?

Reply:
ok.